plugins/samson_ledger/test/lib/samson_ledger/client_test.rb

```diff
@@ -29,10 +40,71 @@
   end
 
   describe ".post_deployment" do
+    before do
+      @asserts = Proc.new {}
```

Comment:
Oh. Forgot to mention an interesting side effect when it crashed - the following fixture file were created automatically if and only if it crashes:

```
test/fixtures/slack_identifiers.yml
test/fixtures/kubernetes/cluster_deploy_groups.yml
test/fixtures/kubernetes/clusters.yml
test/fixtures/kubernetes/deploy_group_roles.yml
test/fixtures/kubernetes/release_docs.yml
test/fixtures/kubernetes/releases.yml
test/fixtures/kubernetes/roles.yml
```